v2/datastream-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToBigQuery.java

```diff
@@ -226,9 +220,7 @@ public interface Options
         order = 8,
         description = "Name or template for the dataset to contain staging tables.",
         helpText =
-            "This is the name for the dataset to contain staging tables. This parameter supports "
-                + "templates (e.g. {_metadata_dataset}_log or my_dataset_log). Normally, this "
-                + "parameter is a dataset name.")
+            "This is the name for the dataset to contain staging tables. This parameter supports templates (e.g. {_metadata_dataset}_log or my_dataset_log). Normally, this parameter is a dataset name. Defaults to: {_metadata_dataset}.")
```

Comment:
```suggestion
            "The name of the dataset that contains staging tables. This parameter supports templates, for example {_metadata_dataset}_log or my_dataset_log. Normally, this parameter is a dataset name. Defaults to: {_metadata_dataset}.")
```

Reply:
Do these values actually have curly braces, or are those supposed to indicate a variable? If the curly braces aren't part of the value, we should remove them.